#! /bin/sh /usr/share/dpatch/dpatch-run
## 90-patch-mpc-basedir.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use /usr/share/ace/bin/MakeProjectCreator path for MPC.

@DPATCH@
--- ACE_wrappers.orig/bin/mwc.pl
+++ ACE_wrappers/bin/mwc.pl
@@ -19,12 +19,7 @@
 use File::Spec;
 use File::Basename;
 
-my($basePath) = $FindBin::Bin;
-if ($^O eq 'VMS') {
-  $basePath = File::Spec->rel2abs(dirname($0)) if ($basePath eq '');
-  $basePath = VMS::Filespec::unixify($basePath);
-}
-$basePath .= '/MakeProjectCreator';
+my($basePath) = '/usr/share/ace/bin/MakeProjectCreator';
 
 my($mpcroot) = $ENV{MPC_ROOT};
 my($mpcpath) = (defined $mpcroot ? $mpcroot :
--- ACE_wrappers.orig/bin/mpc.pl
+++ ACE_wrappers/bin/mpc.pl
@@ -19,12 +19,7 @@
 use File::Spec;
 use File::Basename;
 
-my($basePath) = $FindBin::Bin;
-if ($^O eq 'VMS') {
-  $basePath = File::Spec->rel2abs(dirname($0)) if ($basePath eq '');
-  $basePath = VMS::Filespec::unixify($basePath);
-}
-$basePath .= '/MakeProjectCreator';
+my($basePath) = '/usr/share/ace/bin/MakeProjectCreator';
 
 my($mpcroot) = $ENV{MPC_ROOT};
 my($mpcpath) = (defined $mpcroot ? $mpcroot :
